test/common/odp_cunit_common.c

```diff
@@ -7,13 +8,15 @@
 #include "config.h"
 
 #include <string.h>
+#include <unistd.h>
 #include <odp_api.h>
 #include "odp_cunit_common.h"
 #include <odp/helper/odph_api.h>
 /* Globals */
 static odph_odpthread_t thread_tbl[MAX_WORKERS];
 static odp_instance_t instance;
 
+extern const char *__progname;
```

Comment:
Please use a copy of `argv[0]` instead of progname.

Reply:
save argv[0] in some global variable?

Reply:
@muvarov yes